doc/source/html-styling.ipynb

```diff
@@ -558,14 +587,16 @@
    },
    "outputs": [],
    "source": [
-    "df.style.highlight_min(axis=0)"
+    "df.style.bar(subset=['A', 'B'], color='#d65f5f')"
    ]
   },
   {
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "Use `Styler.set_properties` when the style doesn't actually depend on the values."
+    "New in version 0.20.0 is the ability to customize further the bar chart: You can now have the `df.style.bar` be centered on zero or midpoint value (in addition to the already existing way of having the min value at the left side of the cell), and you can pass a list of `[color_negative, color_positive]`.\n",
```

Comment:
The table below is a nice overview. But it is also a bit hard to interpret as a user what is the actual code he/she should use. Can you add first an example with the existing frame? Eg `df.style.bar(subset=['B'], align='mid')`. And then give the overview table.